src/main/java/seedu/weeblingo/ui/MainWindow.java

```diff
@@ -215,6 +217,21 @@ private CommandResult executeCommand(String commandText) throws CommandException
         }
     }
 
+    private void changePlaceHolderContent(boolean isShowingHistory) {
+        int index = -1;
```

Comment:
What is this index referring to? Would it be better to set it as a static variable at the start instead of a magic number?

Reply:
It is essentially because
```java
int index = -1;
        if (isShowingHistory) {
            index = flashcardListPanelPlaceholder.getChildren().indexOf(flashcardListPanel.getRoot());
            if (index != -1) {
                flashcardListPanelPlaceholder.getChildren().set(index, scoreHistoryListPanel.getRoot());
            }
        }
```
the
```java
indexOf(...)
```
method would return `-1` if the object to look for is not in the `List`.

Thus a static variable seems not very ok but I kind of cannot think of any elegant solution to it...(?:grimacing:)

Reply:
I think there is a contains method?